docs/spec_draft.md

```diff
@@ -582,6 +583,78 @@ Produces a `result` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values from one element type to another
+on `operand` tensor and produces a `result` tensor.
+
+More formally, conversion from element `A` of `operand` to element `B` of
+another type is defined as the following:
+  * Conversion of same element type with different precision:
+    * The result is rounded using any normalizing rounding-direction attributes
+      from the IEEE-754 specification (e.g. `roundTiesToEven`).
+  * Conversion from integer to:
+    * signed integer: signExtend or truncate.
+    * unsigned integer: zero-extend or truncate.
+    * boolean: `B = A != 0 ? true : false`.
+    * floating-point*: `B = (float) A`.
```

Comment:
As per the standard `The roundTiesToEven rounding-direction attribute shall be the default rounding-direction attribute for results in binary formats.` Is this the behavior of HLO as well? If yes then we can write

* floating point
  * `A` is representable in `T'`, B = round(`A`), where `round` implements the `roundToIntegralTiesToEven`.
  * `A` is not representable in `T'`, B  = `+/-infinity`

Reply:
>>  Is this the behavior of HLO as well? 

One way to get the default rounding mode in HLO: find two values x and y such that y > x and one is odd and the other is even.And then try to convert a midway value.
   
https://onlinegdb.com/0ekW_Yc-F